quiz-template.html

```diff
@@ -1,148 +1,189 @@
+
+      #result-category {
+        font-size: larger;
+      }
+
+      fieldset.question {
+        border-width: 0;
+      }
+
+      /* Question headers */
+      fieldset.question legend {
+        font-weight: bold;
+        font-size: larger;
+      }
+    </style>
+
+    <script>
+      /* Global variables*/
+      var totalQuestions = 3;
+      var totalChoices = 3;
+
+      /* Different choice point totals */
+      var xTotal = 0;
+      var yTotal = 0; 
+      var zTotal = 0;
+
+      /* Different choice attributes */
+      var categoryName;
+      var categoryDescription;
+
+      /* Hides the quiz page */
+      function hideMain() {
+        document.getElementById("quiz-main").style.display = "none";
+      }	
+
+      /* Calculates which category a user fits in */
+      function grade() {
+        for(var q = 1; q <= totalQuestions; q++) { // all the questions 
+          var currentQuestionName = "question" + q;
+          var currentQuestions = document.getElementsByName(currentQuestionName);
+
+          for (var c = 0; c < currentQuestions.length; c++) {
+            var option = currentQuestions[c];
+            if (!option.checked) {
+              continue;
             }
-            console.log(radio.value);
-            if(radio.value == "x"){ // category x
-                xTotal += 1;
-            } else if(radio.value == "y"){ // category y
-                yTotal += 1;
-            } else if (radio.value == "z"){ // category z
-                zTotal += 1;
+            // console.log(option.value);
+
+            switch (option.value) {
+              case "x": // category x
+                xTotal++;
+                break;
+              case "y": // category y
+                yTotal++;
+                break;
+              case "z": //category z
+                zTotal++;
+                break;
             }
-		}
-	}
-}
-
-/* Generates the text to appear in the results */
-function generateCategory(){
-	if(xTotal > yTotal && xTotal > zTotal){ // category x
-		categoryName = "X";
-		description = "Supplementary information for x :3"
-	} else if (yTotal > xTotal && yTotal > zTotal){ // category y
-		categoryName = "Y";
-		description = "Supplementary information for y :3"
-	} else if (zTotal > xTotal && zTotal > yTotal){ // catgory z
-		categoryName = "Z";
-		description = "Supplementary information for z :3"
-	} else { // if ties are possible or if no questions are answered
-		categoryName = "Any"
-		description = "Hm maybe you should retake the quiz? :3"	
-	}
-}
-
-/* TODO: 
-- Check whether the user answered all questions.
-- Create a button to reset the quiz. 
-*/
-function displayResult(){
-	grade();
-	generateCategory();
-	hideHome();
-	document.write("<font face = 'Arial'><center>Congratulations! You are in the </font>" + "<br><font size = '14px'>" + categoryName + "</font><br>" + "<font face = 'Arial'>category!" + "<br>" + description + "</center></font>");
-}
-</script>
-
-<body bgcolor="#FFFFFF">

```

Comment:
The `bgcolor` attribute is something from HTML 3.2, possibly older. Don't ever use this.